source/frontend/src/components/propertySelector/MapSelectorContainer.tsx

```diff
@@ -71,9 +71,10 @@ export const MapSelectorContainer: React.FunctionComponent<IMapSelectorContainer
       {activeSelectorTab === SelectorTabNames.list ? (
         <Button
           variant="secondary"
-          onClick={() =>
-            addProperties(searchSelectedProperties, modifiedMapProperties, addSelectedProperties)
-          }
+          onClick={() => {
+            addProperties(searchSelectedProperties, modifiedMapProperties, addSelectedProperties);
+            setSearchSelectedProperties([]);
```

Comment:
resets the selection between clicking the add button. That ensures that previous selections aren't included in future add actions.